src/configureWorkspace/configurePython.ts

```diff
@@ -60,6 +60,20 @@ services:
       dockerfile: Dockerfile
 $ports$`;
 
+const dockerComposeDebugfile = `version: '3.4'
+
+services:
+  $service_name$:
+    image: $service_name$
+    build:
+      context: .
+      dockerfile: Dockerfile
+    entrypoint: /bin/bash
+    command: -c "pip install debugpy -t /tmp && python /tmp/debugpy --wait-for-client --listen 0.0.0.0:5678 $cmd$"
```

Comment:
How big is `debugpy`? There's an implicit disadvantage that other dependencies are installed via the Dockerfile, and thus benefit from image caching, but this one would not.